logserver.c

```diff
@@ -983,6 +983,20 @@ void pv_logserver_toggle(struct pantavisor *pv, const char *rev)
 	}
 }
 
+static void logserver_capture_dmesg()
+{
+	errno = 0;
+	int fd = open("/dev/kmsg", O_RDONLY | O_NONBLOCK);
```

Comment:
this is a good example for me to learn how this works as it is pretty simple :) Where are we closing this?

Reply:
i am not sure if we have to close ita fter sending it to logserver ... logserver definitely closes the fd by itself so .... lets try really and be sure we use this correctly indeed.

Reply:
Currently, the system relies on two ways for closing the fds, the first is the unsubscribe function that function and the second is the detection of fds in bad states. So @anibalportero is right this fd in particular never be close, maybe we will need a function to close and unsubscribe all fds when we want to shut down the system (maybe only in the pv_logserver_stop() function?)